crates/control/src/motion/stand_up_back.rs

```diff
@@ -62,19 +62,15 @@ impl StandUpBack {
     }
 
     pub fn cycle(&mut self, context: CycleContext) -> Result<MainOutputs> {
-        let stand_up_back_estimated_remaining_duration =
-            if let MotionType::StandUpBack = context.motion_selection.current_motion {
-                let last_cycle_duration = context.cycle_time.last_cycle_duration;
-                let condition_input = context.condition_input;
+        let stand_up_back_estimated_remaining_duration = {
+            let last_cycle_duration = context.cycle_time.last_cycle_duration;
+            let condition_input = context.condition_input;
 
-                self.interpolator
-                    .advance_by(last_cycle_duration, condition_input);
+            self.interpolator
+                .advance_by(last_cycle_duration, condition_input);
 
-                Some(self.interpolator.estimated_remaining_duration())
-            } else {
-                self.interpolator.reset();
-                None
-            };
+            self.interpolator.estimated_remaining_duration()
+        };
```

Comment:
This does not look right?! You somehow miss the reset now?

Reply:
Reset has returned